docs/gpu-support.md

```diff
@@ -0,0 +1,73 @@
+# Deploying with Accelerators in the Cloud HPC Toolkit
+
+## Supported modules
```

Comment:
This makes it sound like only the following modules support GPUs. 
Doesn't Batch, HTCondor or Omnia support GPUs?

Reply:
I think it would still be important to mention that Batch supports it...

Reply:
Added